converter/converting.py

```diff
@@ -374,14 +373,23 @@ def data_objects_from_magetab(idf_file_path, sdrf_file_path):
         assay_data = AssayData.from_magetab(name, file_objects, group)
         ad_objects.append(assay_data)
 
+    # Analysis (processed data)
+    print(processed_data)
+    analysis_objects = []
+    for f_name, f_attrib in processed_data.items():
```

Comment:
Possibly use a list comprehension as you've done elsewhere?

Reply:
Yes, could do that in a list comprehension which is generally preferred and faster. But it does get a bit long and I am not sure it makes is more readable:
analysis_objects = [Analysis.from_magetab([DataFile.from_magetab(f_attrib)], f_attrib) for f_attrib in processed_data.values()]